File: yieldoncost.py
print('-=-' * 20)
print('Calculando Yield on Cost (YOC) de FIIs')
print('-=-' * 20)

preco_aquisicao = float(input('Qual seu custo médio de aquisição? ').replace(',', '.'))
preco_aquisicao * 100
rendimento = float(input('Quanto rendeu no último mês? ').replace(',', '.'))
rendimento * 12
yield_on_cost = rendimento / preco_aquisicao * 100

print('Seu yield on cost foi de {:.3f}% nesse mês.'.format(yield_on_cost))
print('A rentabilidade anualizada seria {:.3f}%.'.format(yield_on_cost * 12))

File: assets/js/yieldoncost.js
const calcularYieldOnCost = () => {
  let precoAquisicao = document
    .getElementById("precoAquisicao")
    .value.replace(/,/g, ".");
  precoAquisicao * 100;
  let rendimento = document
    .getElementById("rendimento")
    .value.replace(/,/g, ".");
  rendimento * 12;

  let yieldOnCost = (rendimento / precoAquisicao) * 100;

  let resultadoMes = document.querySelector("section#resultadoMes");
  let resultadoAno = document.querySelector("section#resultadoAno");

  resultadoMes.innerHTML = `<p>Seu yield on cost foi de ${yieldOnCost.toFixed(
    3
  )}% nesse mês.</p>`;
  resultadoAno.innerHTML = `<p>A rentabilidade anualizada seria de ${(
    yieldOnCost * 12
  ).toFixed(3)}%.</p>`;
};
